README.md

```diff
@@ -1,12 +1,19 @@
-# Nuclear Speckle Image Analysis
+# Nuclear Speckle Image Profiling
 
-In this repository, image analysis and image-based profiling is performed on a nuclear speckle dataset to extract CellProfiler morphology features to predict nuclear speckle protein markers.
+In this repository, image analysis and image-based profiling is performed on a nuclear speckle dataset to extract CellProfiler morphology features for predicting nuclear speckle protein markers.
```

Comment:
```suggestion
In this repository, image analysis and image-based profiling are performed on a nuclear speckle dataset to extract CellProfiler morphology features for predicting nuclear speckle protein markers.
```

Reply:
Also consider clarifying what is being predicted. Is the number of nuclear speckle being predicted, or the identity of the protein, or the signal of the protein?

Reply:
This is a great question! We are predicting the morphology of the nuclear speckle proteins using two different methods: traditional regression and a deep learning method. 

I will update this in the README now.